src/odfesqlodbc/es_communication.cpp

```diff
@@ -87,6 +115,22 @@ void ESCommunication::AwsHttpResponseToString(
     output.assign(buf.data(), avail);
 }
 
+void ESCommunication::PrepareCursorResult(ESResult& es_result) {
+    // Prepare document and validate schema
+    try {
+        LogMsg(ES_DEBUG, "Parsing result JSON with cursor.");
+        es_result.es_result_doc.parse(es_result.result_json,
```

Comment:
Does this just do validation of the cursor result?

Reply:
validates and update doc with parsed datarows